schemas/2015-05-21-preview/Microsoft.DevTestLabs.json

```diff
@@ -0,0 +1,59 @@
+{
+  "id": "http://schema.management.azure.com/schemas/2015-08-01/Microsoft.DevTestLab.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.DevTestLabs",
+  "description": "Azure DevTest Labs Resource Types",
+  "definitions": {
+    "labs": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "enum": [
+            "Microsoft.DevTestLabs/labs"
+          ]
+        },
+        "apiVersion": {
+          "enum": [
+            "2015-05-21-preview"
+          ]
+        }
+      }
+    },
+    "environments": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "enum": [
+            "Microsoft.DevTestLabs/environments"
+          ]
+        },
+        "apiVersion": {
+          "enum": [
+            "2015-05-21-preview"
+          ]
+        },
+         "properties": {
+            "labId": {"type": "string"},
+            "vms": {
+                "type": "array",
+                "items": {
+                    "type": "object",
+                    "properties": {
+                        "name": {"type": "string"},
+                        "vmTemplateName": {"type": "string"},
+                        "size": {"type": "string"},
+                        "userName": {"type": "string"},
+                        "password": {"type": "string"},
+                        "sshKey": {"type": "string"},
+                        "isAuthenticationWithSshKey": {"type": "boolean"}
+                    }
```

Comment:
Please also add "required" part for the "items" if there is any required property.